orbit/pkg/update/hash.go

```diff
@@ -12,29 +12,34 @@ import (
 	"github.com/theupdateframework/go-tuf/data"
 )
 
-// CheckFileHash checks the file at the local path against the provided hash
-// functions.
-func CheckFileHash(meta *data.TargetFileMeta, localPath string) error {
-	hashFunc, hashVal, err := selectHashFunction(meta)
+// checkFileHash checks the file at the local path against the provided hash functions.
+func checkFileHash(meta *data.TargetFileMeta, localPath string) error {
```

Comment:
The functions in this file could use some unit tests. I know, I know, I wrote them and didn't make unit tests...